pkgs/applications/networking/modem-manager-gui/default.nix

```diff
@@ -15,15 +15,15 @@
 , ninja
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "modem-manager-gui";
   version = "0.0.20";
 
   src = fetchFromGitLab {
     domain = "salsa.debian.org";
     owner = "debian";
     repo = "modem-manager-gui";
-    rev = "upstream%2F${version}";
+    rev = "upstream%2F${finalAttrs.version}";
     sha256 = "1pjx4rbsxa7gcs628yjkwb0zqrm5xq8pkmp0cfk4flfk1ryflmgr";
```

Comment:
is this one not changed because hash is not in SRI format ?

Reply:
Missed it when changing the others, fixed it in https://github.com/NixOS/nixpkgs/pull/275748/commits/eff4440d59de63e8c774ad7f01961ed1baec4f3a